src/proxy_handler.cr

```diff
@@ -107,6 +107,21 @@ class Mitm::ProxyHandler
   # be created. Target parameters are supplied with the websocket already
   # connected to the client.
   def open_websocket(host, resource, port, tls, headers, client_ws)
+
+    # inform server that client doesn't support permessage-deflate compression
+    if headers.try &.includes_word?("Sec-WebSocket-Extensions", "permessage-deflate")
+      Log.debug {"Removing permessage-deflate from Sec-WebSocket-Extensions header"}
```

Comment:
The same for this one.

Reply:
let's keep it for now, it will be rare message.